src/components/ChangeInfoComponent.vue

```diff
@@ -0,0 +1,88 @@
+<script setup lang="ts">
+import CardComponent from "@/components/CardComponent.vue";
+import { useField } from "vee-validate";
+import apiService from "@/services/ApiService";
+import { useAuthStore, useUserStore } from "@sudosos/sudosos-frontend-common";
+import { useToast } from "primevue/usetoast";
+import { useI18n } from "vue-i18n";
+
+const authStore = useAuthStore();
+const toast = useToast();
+const userStore = useUserStore();
+const { t } = useI18n();
+
+let isLocal = false;
+if(userStore.getCurrentUser.user) {
+  isLocal = (userStore.getCurrentUser.user.type == "LOCAL_USER");
+}
+
+
+const { value: inputFirstName, errorMessage: inputFirstNameError } = useField('inputFirstName', validateFirstName);
+const { value: inputLastName, errorMessage: inputLastNameError } = useField('inputLastName', validateLastName);
+const { value: inputEmail, errorMessage: inputEmailError } = useField('inputEmail', validateEmail);
+
+
+function validateFirstName(checkFirstName: string) {
```

Comment:
For this we don't need custom validators. See [yup](https://github.com/jquense/yup#schema-defaults).